mobile/lib/features/wallet/balance_row.dart

```diff
@@ -53,6 +60,17 @@ class _BalanceRowState extends State<BalanceRow> with SingleTickerProviderStateM
       rowBgColor = theme.lightning;
       icon = SvgPicture.asset("assets/Lightning_logo.svg");
       amount = walletChangeNotifier.lightning();
+    } else if (widget.walletType == WalletHistoryItemDataType.stable) {
+      final positionBtc = positionChangeNotifier.positions[ContractSymbol.btcusd];
+      final hasStable = positionBtc != null && positionBtc.direction == Direction.short;
```

Comment:
I haven't looked much into how stabilized sats are implemented here - would this mean that any short against bitcoin (even one that isn't semantically 'stabilizing sats' but just a regular bet) would show up as stablesats? Or is this not the case

Reply:
Yes at the moment we simply display a short position leverage 1 as synthetic usd.

Reply:
good catch, this one should be abstracted away into `positionChangeNotifier` too. 
Having business logic in widget code was not a good idea 😔

Reply:
fixed.

Reply:
> Yes at the moment we simply display a short position leverage 1 as synthetic usd.

Where should we discuss this UX further? @holzeis